ios/cocoapods_processes.md

```diff
@@ -0,0 +1,66 @@
+# Cocoapods Processes
+
+We like to make reusable code and cocoapods is a great way to distribute that code. Individual projects may be making their own pods, but we also maintain the `Intrepid` pod within the entire iOS team. This guide aims to cover the necessary steps for creating your own pod and also maintaining one, like we must do with the `Intrepid` pod.
+
+The goal of this doc is not to copy all the info available at [cocoapods.org](https://cocoapods.org/). This is specifically oriented towards helpful advice specific to Intrepid processes.
+
+## Building Your Own Pod
+
+The first thing to do is just start a new xcode project that holds all of your source components. There will really be two parts of your project. There will be the actual pod components that you are building to be reusable by others -- let's keep these in a folder called `Source`. Then there will be the rest of your project that should really act like a demo of the pod.
+
+Your structure could end up looking something like this:
+
+![pod folder structure](./images/pod_folder_setup.png)
+
+Then you'll want to make a podspec file. You can start by just copying the text below and customizing. This should be saved in the top level of your project directory and be named like `[Your-pod-name].podspec`.
+
+```ruby
+Pod::Spec.new do |spec|
+  spec.name         = 'MyPodName'
+  spec.version      = '0.0.1'
+  spec.license      = 'NO LICENSE'
```

Comment:
For pods we're just making for internal use, I think `no license` is the way to go, but if we're making it open source, we should use MIT. I'll just add a comment to use MIT for open source pods.